### DISCUSSION TITLE
Error with laravel steam auth

### DISCUSSION THREAD

**Discussion by Eli0uce** [ilzrv/laravel-steam-auth#13] [Q&A]
Hello i would like use your script , i have test but it doesnt work, can you help me? you can contact me on discord : eliouce

**Comment by ilzrv** [comment]
Hello! You can post your question right here

**Comment by Eli0uce** [comment]
ok, when i click on my button to login with steam, i have this error, but i have only one SteamAuthController.
![image](https://github.com/ilzrv/laravel-steam-auth/assets/93868767/5be36aa9-9b37-4928-aad3-08f9b89035fd)
![image](https://github.com/ilzrv/laravel-steam-auth/assets/93868767/20a1dfdd-1864-463c-818d-6d532c1a55a3)
![image](https://github.com/ilzrv/laravel-steam-auth/assets/93868767/c8a8af03-49bb-40f4-bd36-687ca53bffb5)
